boards/arduino_mkrvidor4000/examples/run_fpga.rs

```diff
@@ -0,0 +1,30 @@
+#![no_std]
+#![no_main]
+
+extern crate arduino_mkrvidor4000 as hal;
+
+use hal::clock::GenericClockController;
+use hal::delay::Delay;
+use hal::entry;
+use hal::gpio::IntoFunction;
+use hal::pac::{CorePeripherals, Peripherals};
+use hal::prelude::*;
+
+#[entry]
+fn main() -> ! {
+    let mut peripherals = Peripherals::take().unwrap();
+    let core = CorePeripherals::take().unwrap();
+    let _clocks = GenericClockController::with_external_32kosc(
+        peripherals.GCLK,
+        &mut peripherals.PM,
+        &mut peripherals.SYSCTRL,
+        &mut peripherals.NVMCTRL,
+    );
+    let mut pins = hal::Pins::new(peripherals.PORT);
+
+    // Enable 48MHZ clock output for FPGA
+    // https://github.com/arduino/ArduinoCore-samd/blob/master/variants/mkrvidor4000/variant.cpp#L229
+    let _fpga_clk: hal::gpio::Pa27<hal::gpio::PfH> = pins.gclk.into_function(&mut pins.port);
```

Comment:
this should be .into_function_h()

Reply:
Ok cool, that makes the example cleaner. 
Is there a similar thing for into_pad? For the i2c battery charging example, I had to fully specify the types because it couldn't be inferred.

Reply:
No, I don't think so.